user_manual/pim/calendar.rst

```diff
@@ -53,27 +49,31 @@ previous calendar, creating a new calendar is the way you should go.
 3. After clicking on the checkmark, your new calendar is created and can be
    synced across your devices, filled with new events and shared with your friends
    and colleagues.
+   
+.. figure:: images/calendar_create.gif
 
 Edit, Download or Delete a Calendar
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Sometimes you may want to change the color or the entire name of a previous
 imported or created calendar. You may also want to export it to your local
-hard drive or delete it forever.
+hard drive or delete it forever:
 
-.. note:: Please keep in mind that deleting a calendar is a irreversible action.
-          After deletion, there is no way of restoring the calendar unless you
-          have a local backup.
+1. Click on the three-dot-menu of the respective calendar.
+
+2. Click on ``Edit name``, ``Download`` or ``Delete``:
 
 .. figure:: images/calendar_dropdown.png
             :scale: 50%
 
-1. Click on the three-dot-menu of the respective calendar.
+3. Click on ``Edit color``:
 
 .. figure:: images/calendar_editing.png
             :scale: 50%
 
-2. Click on ``Edit``, ``Download`` or ``Delete``.
+.. note:: Please keep in mind that deleting a calendar is a irreversible action.
```

Comment:
a**n** irreversible ?

Reply:
This doesn't make it better. I think you can correct an obvious typo without doing a separate pull request...

Reply:
Typo corrected 😉.